hangupsbot/sync/__init__.py

```diff
@@ -131,6 +131,10 @@
     'sync_format_reply_empty': '| <i><b>{name}</b>: {image_tag}~</i>\n',
     'sync_format_reply_bot': '| <i>{image_tag}{text}</i>\n',
     'sync_format_reply_bot_empty': '| <i>{image_tag}~</i>\n',
+
+    # automatically restrict TG users on join, possible values are: messages,
+    # media, sticker, websites, sticker+websites
+    'sync_restrict_user': '',
```

Comment:
Let's keep the telesync defaults in `plugins.telesync.setup_config.default_config` and access them with `bot.get_config_suboption`.